iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -512,6 +512,23 @@ def _getitem_user(self, key: str) -> typing.Any:
 
         raise KeyError(f"User defined property not found: {key}")
 
+    def get(
+        self,
+        key: str,
+        *args: typing.Any
+    ) -> typing.Any:
+        """Return the config value or its given default."""
+        if len(args) > 1:
```

Comment:
```python
>>> len(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```

Reply:
yeah, args is an array…

Reply:
Due to the `*args` notation the `args` value is always a list but may be empty.